### PR TITLE
Added default global variables

### PR DESCRIPTION
**Pull request by andrewssobral**

### CODE REVIEW

File: config/scheduler/global_variables.xml [hunk 1]
```diff
@@ -2,20 +2,41 @@
...
         xmlns="urn:proactive:globalvariables:3.13" xsi:schemaLocation="urn:proactive:globalvariables:3.13 http://www.activeeon.com/public_content/schemas/proactive/globalvariables/3.13/globalvariables.xsd">
+        <!-- Multiple filters can be added, to apply different set of global variables to different workflows -->
         <filter>
             <select>
                 <!-- Xpath expressions are used to filter workflows affected by global variables.
                 Example: to select workflows belonging to the basic-examples bucket
                 <xpath><![CDATA[/job/genericInformation/info[@name='bucketName' and @value='basic-examples']]]></xpath>
                 -->
-                <xpath><![CDATA[.]]></xpath>
+                <!-- <xpath><![CDATA[.]]></xpath> -->
```
Comment (fviale) [inline config/scheduler/global_variables.xml:12]: can you please put all this in a separate filter ?

I would like to keep the default empty filter

**Comment by fviale** [comment]
Retest this please

File: config/scheduler/global_variables.xml [hunk 2]
```diff
@@ -2,20 +2,41 @@
...
+                <xpath><![CDATA[(/job/genericInformation/info[@name='pca.states' and @value='(VOID,RUNNING)']) or not (/job/genericInformation/info[@name='pca.states'])]]></xpath>
                 <!-- more than one xpath expression can be added. In that case, global variables are applied when all xpath expressions match -->
             </select>
             <!-- add global variables or generic information below -->
             <variables>
+                <variable name="NATIVE_SCHEDULER" value="" advanced="true" description="Name of the Native Scheduler node source to use when the workflow tasks must be deployed inside a cluster such as SLURM, LSF, etc." group="Resource Management" hidden="false"/>
```
Comment (fviale) [inline config/scheduler/global_variables.xml:18]: you can use the same model as for NODE_SOURCE_NAME

the semantic is to choose a node source

**Comment by fviale** [comment]
Retest this please

**Comment by andrewssobral** [comment]
Retest this please

**Comment by andrewssobral** [comment]
This PR was merged together with https://github.com/ow2-proactive/proactive-examples/pull/1131

**Pull request merged**